feature/account/src/commonMain/kotlin/org/mifos/mobile/feature/account/clientAccount/screens/ClientAccountsScreen.kt

```diff
@@ -23,37 +24,40 @@ import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.platform.LocalContext
-import androidx.compose.ui.res.stringResource
-import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
-import com.google.accompanist.pager.rememberPagerState
+import mifos_mobile.feature.account.generated.resources.Res
+import mifos_mobile.feature.account.generated.resources.feature_account_loan
+import mifos_mobile.feature.account.generated.resources.feature_account_loan_account
+import mifos_mobile.feature.account.generated.resources.feature_account_savings
+import mifos_mobile.feature.account.generated.resources.feature_account_savings_account
+import mifos_mobile.feature.account.generated.resources.feature_account_share
+import mifos_mobile.feature.account.generated.resources.feature_account_share_account
+import org.jetbrains.compose.resources.stringResource
+import org.koin.compose.viewmodel.koinViewModel
 import org.mifos.mobile.core.common.Constants
-import org.mifos.mobile.core.designsystem.components.FloatingActionButtonContent
-import org.mifos.mobile.core.designsystem.components.MifosScaffold
-import org.mifos.mobile.core.designsystem.components.MifosTabPager
-import org.mifos.mobile.core.designsystem.icons.MifosIcons
+import org.mifos.mobile.core.designsystem.component.FloatingActionButtonContent
+import org.mifos.mobile.core.designsystem.component.MifosScaffold
+import org.mifos.mobile.core.designsystem.component.MifosTabPager
+import org.mifos.mobile.core.designsystem.icon.MifosIcons
 import org.mifos.mobile.core.designsystem.theme.MifosMobileTheme
-import org.mifos.mobile.core.model.entity.CheckboxStatus
 import org.mifos.mobile.core.model.enums.AccountType
-import org.mifos.mobile.core.ui.utils.DevicePreviews
-import org.mifos.mobile.feature.account.R
+import org.mifos.mobile.core.ui.utils.DevicePreview
 import org.mifos.mobile.feature.account.account.screens.AccountsScreen
 import org.mifos.mobile.feature.account.clientAccount.utils.ClientAccountFilterDialog
 import org.mifos.mobile.feature.account.clientAccount.utils.ClientAccountsScreenTopBar
+import org.mifos.mobile.feature.account.utils.CheckboxStatus
 import org.mifos.mobile.feature.account.viewmodel.AccountsViewModel
 
 @Composable
 internal fun ClientAccountsScreen(
+    isNetworkConnected: Boolean,
```

Comment:
Inject NetworkMonitor directory in AccountsViewModel and observe as AccountsViewModel in here and composable and use